`enclave_test_private.pem` is a DEBUG ONLY private key for signing test
enclaves. It is generated and included in the linux_sgx patch file only for
reproducible build purposes. A cautious user may generate another one to replace
it in their Bazel cache or modify the patch file itself to apply a different PEM
file. The PEM in this patch was generated by the following command:

    openssl genrsa -3 -out enclave_test_private.pem 3072
--
--- /dev/null
+++ enclave_test_private.pem
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIG4gIBAAKCAYEAvxHHEUZUevlX8X7GelXrTRhWuJQ3btJyuFpS9bRMu5cXsOVs
+Sk/6+rKySch9ilqNYiLrtpxqgYgULtdL2Qm83HBCFUwCL7Psw61payx9nByYDcUn
+jfhNC4TkBT+Klj5rgHdxO5QRwAL6e+iVOxUVA0kZ+HJKRua+ff26joXrjTInZFRN
+8Rdfr7BVzh5KBTTAuOnlOlj3NIwjjGT+xSSvj1FSOkXIuk7tKg/LiNcc6EaNAmNE
+P2/LYWX1UKiRPWlndt82djhqPfqQ7aMkph9ttE76WA67NEnzo3tcfDj9UvAfuxdw
+XR1vZKhXtbzB7JI64OoSaKjkbkgNkgs3ymZsuHKIIbWBJp1vkIZju2wrpyNwU1Si
+rmyoLxLkWZvwQ8i40BGLBCYIJ1Pgt/pPNXeB7O/cxdch2pMrZY9trAkhmDeM2DEX
+O6W+M5kL7CM+i0Dk+ZLwQJt0aUHhqyUD3I/FisHqpA36xFCdvjFSu60Brc2mvAtE
+y8Iagnwn8xn2hOY7AgEDAoIBgH9hL2DZjadQ5Uup2abj8jNljyW4JPSMTHrm4fki
+3dJkunXuSDGKp1HMdtva/lw8XkFsnSRoRwEFYsnk3Ttb0z2gLA4yrB/NSIJzm5zI
+U71oZV6Db7P63get7VjVBw7UR6r6S30NYSqspv1FuNIODgIwu/r23C9Efv6pJwmu
+nQjMGkLi3qC6P8p1jolphq4jKyXxQ3w7T3hdbQhDVINtyl+LjCbZMHw0nhwKh7CP
+aJrZs1bs2CpKh5ZD+OBwYNObmSXc8pdqyUXVG6mAtHwZIB3NuyZONZLYeyW6+m49
+d5aC0wlx3vuI78Jasll/2CwvD6ZH7wXaX3hWaODlPKJ/7W1Gk3ohIs1yS4SsSyHz
+P033Qb3fEBJeVvLp2cRupXcOvShu+FQPsqlj8no+TiJZkFz26DAKsS3PAYcc/GgD
+3QOQGnov3plYYFFmGDDWh7neR6Vfd0ycAnzeHgU2nMPT0XHsCYP1fd6ZZTQgOwfX
+TiExJwRRNenqwy2W1KlTmWWmOwKBwQD75B+QTpG3IAWtYO6HOdkLw+RtKvU6HVbb
+srDwts/EU9mwhklLAcfQpVwrf4rYL+oEmL2VEX5uFYFubFLJuqeDwMiLQnKAkfmz
+sKPJZtmp7i6uBRJPSit0SRtbgPWpM2sdB7vRZja79NxBinLExtI6NCUeKLzT919p
+Bst4HLedem4aNLaxV6PixMjFLceHssFJrmkTGd0GaDY3uOEL/ulEpvzO3CYMAQMr
+7f7HvqiL/wyoyd8nMDkaw2eOZ4Q+rzUCgcEAwi+rAsmqnhrhwgEnZL/ke9Z9MW51
+nefkDzAz5iX6LNgJfGZYmM4/8HrwA/1y0hu5YuVuSs/Wy7Du0kcKDOvhEMfiX0Fa
+ceAEldNPPxhzCEJfCID4Q5T6TmsCV3UCZ/8x9mNatKhFbUkYvxJnPSpXJ0lbX55b
+QeVJsd0nl+aedWT3BrSSpEmJ1rse3bOrbMC4N9VUOpeTp4x+a5o3qtMiGzjfj0GX
+6mmCxBL7aCtj8ATfjMeUxbG8o1aBZrUMLb2vAoHBAKftamA0YSTAA8jrSa975gfX
+7Z4co3wTjz0hy0skioLikSBZhjIBL+BuPXJVBzrKnAMQfmNg/vQOVkmdjIZ8b60r
+MFzW9wBhUSJ1woZEkRv0Hx6uDDTcHPgwvOerTnDM8hNafTZEJH1N6CuxodiEjCbN
+bhQbKI1PlPCvMlATJROm9BF4ecuPwpct2y4ehQUh1jEe8Ldmk1mazs/Qlgf/Ri3E
+qInoGV1WAh1JVIUpxbKqCHCGlMTK0LyCRQmaWCnKIwKBwQCBdRyshnG+vJaBVhpD
+KphSjv4g9E5pRUK0ys1EGVFzOrD9mZBl3tVK/KACqPc2vSZB7kmHNTndIJ822gaz
+R+tghUGU1jxL6q25N4oqEEywLD9bAKWCY1GJnKw6TgGaqiFO7OcjGtjzhhB/YZoo
+xuTE25I/vueBQ4Z2k2+6mb747foEeGHC27E50hSTzRzzKyV6jjgnD7fFCFRHvCUc
+jMFnez+01mVG8QHYDKeax5f1WJUIhQ3ZIShs5FZEeLLJKR8CgcBMJ6Jepigoy9ss
+akOAIsJUcIrKWxNguxJBR3TA9FB2FNC0+dYtnJs8NSiU5nRJVbJEOddYXccSNgVJ
+8+pWB91VjOjmfbJqKFWE3V0uyzWMNut223Qi3bmMrURTPyznFMgiEs2hALW6gZyF
+ylDthwEu2SKLotB1cy0gmAc9269RpjaEaNHuTQJLedSbsPp33McvOdsB8V3UUM6M
+VeN+o2L2w5Rp2kUcb2QR13sC/VYJ1BQICvWrScpNfNGnPAKFglU=
+-----END RSA PRIVATE KEY-----
